examples/asr/asr_chunked_inference/ctc/speech_to_text_buffered_infer_ctc.py

```diff
@@ -89,12 +100,18 @@ class TranscriptionConfig:
     # Recompute model transcription, even if the output folder exists with scores.
     overwrite_transcripts: bool = True
 
+    # decoder type for hybrid model could be None for ctc model and ctc for hybrid model
+    decoder_type: Optional[str] = None
```

Comment:
As this script is just for CTC, do we need this param?

Reply:
good point. This can be removed